FHEM/14_SD_WS.pm

```diff
@@ -661,7 +660,7 @@ sub SD_WS_Parse($$)
           my $rawtemp1  = SD_WS_binaryToNumber($_[1],20,23);
           if ($rawtemp100 > 9 || $rawtemp10 > 9 || $rawtemp1 > 9) {
             Log3 $iohash, 3, "$name: SD_WS_Parse $model ERROR - BCD of temperature ($rawtemp100 $rawtemp10 $rawtemp1)";
-            return "";
+            return;
```

Comment:
Bist Du absichtlich von einem leeren Scalar auf undef gegangen?

Reply:
Ja, das ist Absicht, da in der weiteren Verarbeitung sowieso immer eine Prüfung "if (defined($temp)" erfolgt.